### PR TITLE
Add analytics policy

### PR DESCRIPTION
**Pull request by fainashalts**
This PR adds an analytics tracking policy. It is very simple just to inform users of what it means when they enable analytics.

### CODE REVIEW

**Comment by tenthirtyone** [comment]
I'd like to hop on to review because I haven't done much with the site yet - but don't let me block or stop another for reviewing who has greater context.

File: custom_theme/analytics.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            Ganache, and Drizzle), we have added an optional new analytics feature. By default it is turned off but if
+            you enable it, the Truffle developers will receive anonymous information about your version number, the
+            commands you run, and whether commands succeed or fail.
+          </p>
+          <p class="c0"><span class="c5 c2 c7 c1"></span></p>
+          <p class="c9">We don't use this anonymous information for anything other than to find out how we can make
+            Truffle better and have also
+            ensured that it doesn't make for a slower experience by sending this information in a background process.
+          </p>
+          <p class="c0">To turn this feature, run:
```
Comment (davidmurdoch) [inline custom_theme/analytics.html:62]:
```suggestion
          <p class="c0">To enable anonymous analytics, run:
```

File: custom_theme/analytics.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+          <pre class="bd-milk-chocolate"><code>truffle config --enable-analytics</code></pre>
+          </p>
+          <p class="c0">If you want to turn the analytics off, run:
+          <pre class="bd-milk-chocolate"><code>truffle config --disable-analytics</code></pre>
+          </p>
+          <p class="c9">You can also enable and disable analytics when using Truffle Dashboard. Click on the settings
```
Comment (davidmurdoch) [inline custom_theme/analytics.html:68]: Can you link to Truffle Dashboard docs here?

File: custom_theme/analytics.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            icon at the bottom right to make that choice.</p>
+          <p class="c0"><span class="c5 c2 c7 c1"></span></p>
+          <p class="c9">Feel free to go take a peek at the two current places in the code where metrics are
```
Comment (davidmurdoch) [inline custom_theme/analytics.html:71]: Can you link `the code` to the Truffle repo here?

File: custom_theme/analytics.html [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+            gathered: when running a command and to report version and errors. If you go ahead and do a good ol' GitHub
```
Comment (davidmurdoch) [inline custom_theme/analytics.html:72]:
```suggestion
            gathered: when running a command and to report the version used and errors. If you go ahead and do a good ol' GitHub
```

**Review by davidmurdoch — CHANGES_REQUESTED** [review]

File: custom_theme/analytics.html (hunk 2 above)
Comment (fainashalts) [reply custom_theme/analytics.html:68]: done!

File: custom_theme/analytics.html (hunk 3 above)
Comment (fainashalts) [reply custom_theme/analytics.html]: done!

**Comment by fainashalts** [comment]
@davidmurdoch sorry for the delay on this, it kept falling off my radar! Should be good to go now.

**Comment by fainashalts** [comment]
> Hey @fainashalts. Can you resolve the footer conflict?

Resolved!

**Review by davidmurdoch — APPROVED** [review]

**Pull request merged**